### PR TITLE
win: make the toasts appear longer

### PR DESCRIPTION
**Pull request by requilence**
Toast duration was 'short' by default, according to measurements, it is approx 5sec.
Now I set it to 'long' and it stays for 25sec

### CODE REVIEW

**Comment by requilence** [comment]
@soupdiver you are right, I renamed the the PR and set the description

**Comment by soupdiver** [comment]
That's why I asked to _always_ have a description in the PR ;)

File: cmd/cagent/toast_windows.go [hunk 1]
```diff
@@ -30,9 +30,10 @@ func getExecutablePath() string {
 
 func sendErrorNotification(title, message string) error {
 	msg := toast.Notification{
-		AppID:   toastAppID,
-		Title:   title,
-		Message: message,
+		AppID:    toastAppID,
+		Title:    title,
+		Message:  message,
+		Duration: toast.Long,
```
Comment (soupdiver) [inline cmd/cagent/toast_windows.go]: Add a shirt comment which effect `toast.Long` has

File: cmd/cagent/toast_windows.go [hunk 2]
```diff
@@ -48,10 +49,11 @@ func sendErrorNotification(title, message string) error {
...
 func sendSuccessNotification(title, message string) error {
 	msg := toast.Notification{
-		AppID:   toastAppID,
-		Title:   title,
-		Message: message,
-		Actions: []toast.Action{},
+		AppID:    toastAppID,
+		Title:    title,
+		Message:  message,
+		Duration: toast.Long,
```
Comment (soupdiver) [inline cmd/cagent/toast_windows.go]: Add a shirt comment which effect `toast.Long` has

Comment (requilence) [reply cmd/cagent/toast_windows.go]: done

File: cmd/cagent/toast_windows.go (hunk 1 above)
Comment (requilence) [reply cmd/cagent/toast_windows.go]: done

**Pull request merged**